components/wpt-results.html

```diff
@@ -224,9 +229,22 @@
           this._refreshDisplayedNodes()
         }
 
-        const testFileResults = await Promise.all(this.testRuns.map(testRun => {
-          return this._fetchResults(testRun.results_url)
-        }))
+        const testRuns = await Promise.all(
+          this.testRunSpecs
+            .map(async testRun => {
```

Comment:
nit: One line `this.testRunSpecs.map(async testRun => {` to save on indents.

Reply:
Done.